iothub_client/samples/iothub_client_c2d_streaming_proxy_sample/iothub_client_c2d_streaming_proxy_sample.c

```diff
@@ -244,15 +244,15 @@ static UWS_CLIENT_HANDLE create_websocket_client(DEVICE_STREAM_C2D_REQUEST* stre
     size_t host_length;
     const char* path;
     size_t path_length;
-    int port;
+    size_t port;
 
     char host_address[1024];
     char resource_name[1024];
 
     ws_url = ws_url_create(stream_request->url);
```

Comment:
>ws_url_create [](start = 13, length = 13)

There are some issues with these ws_url, not only the port is size_t but the gets for both host and path are guessing the buffer size.  Fix it does not need to be part of this PR but we should open a issue to fix it in the future.

Reply:
Filed the issue to track it.
https://github.com/Azure/azure-c-shared-utility/issues/404